client/src/pages/Profiles/useStyles.ts

```diff
@@ -0,0 +1,78 @@
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles(() => ({
+  flexContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    // backgroundColor: 'red',
+    height: '100%',
+    width: '100%',
+  },
+  header: {
+    marginTop: '30px !important',
+    fontWeight: 'bold !important',
+  },
+  flexRowContainer: {
+    display: 'flex',
```

Comment:
Most of these styles can be added to `Box` component directly

Ditto where applicable!

Reply:
@natey37 Please address comment above